manifests/w/Wesang/Yogiyo/4.0.6/Wesang.Yogiyo.locale.en-US.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Wesang.Yogiyo
+PackageVersion: 4.0.6
+PackageLocale: en-US
+Publisher: Wesang Co.,Ltd.
```

```suggestion
Publisher: Yogiyo
```

It would be better configuring here to match ARP Entries.